#SAK 12758 - FCK editor fix to readd attachment
=svnm= https://source.sakaiproject.org/svn/reference/trunk@45524 reference

#Fill in the SAK or something unique, leave it like this if theres no real patch
--- /dev/null
+++ SAK-12758.tmp
@@ -0,0 +1 @@
+ 
